lib/Select.js

```diff
@@ -125,14 +125,19 @@ module.exports = kind(
 	* @private
 	*/
 	rendered: kind.inherit(function (sup) {
-		return function() {
+		return function () {
 			sup.apply(this, arguments);
 			//Trick to force IE8 onchange event bubble
-			if(platform.ie == 8){
+			if (platform.ie == 8) {
 				this.setAttribute('onchange', dispatcher.bubbler);
 			}
-			this.selectedChanged();
-			this.valueChanged();
+			// This makes Select.selected a higher priority than Option.selected but ensures that
+			// the former works at create time
+			if (this.selected !== 0) {
+				this.selectedChanged();
+			} else {
+				this.updateSelectedFromNode();
```

Comment:
Hi @ryanjduffy, when the conditional is not met, we have a case where `this.selected` is `0` and the value is not subsequently set, because `selectedChanged` is not run due to the `selectedIndex` property of the control also being `0`.